tsfile/src/main/java/org/apache/iotdb/tsfile/write/chunk/ChunkBuffer.java

```diff
@@ -99,21 +99,18 @@ public int writePageHeaderAndDataIntoBuff(ByteBuffer data, int valueCount,
     }
     this.maxTimestamp = maxTimestamp;
     int uncompressedSize = data.remaining();
-    int maxSize = compressor.getMaxBytesForCompression(uncompressedSize);
+    compressor.getMaxBytesForCompression(uncompressedSize);
```

Comment:
Why do you get something without a receiver?

Reply:
In the primary code, the result `maxSize` is unused. I guess this line is to invoke `getMaxBytesForCompression` to perform something(e.g. skipping an unconcerned byte), but not interested in the result.
Maybe @jixuan1989 could explain it more?

Reply:
In that case, I would recommend the method renamed.